src/views/Farms/Farms.tsx

```diff
@@ -192,16 +174,73 @@ const Farms: React.FC = () => {
     setQuery(event.target.value)
   }
 
-  let farmsStaked = []
-  if (isActive) {
-    farmsStaked = stakedOnly ? farmsList(stakedOnlyFarms) : farmsList(activeFarms)
-  } else {
-    farmsStaked = stakedOnly ? farmsList(stakedInactiveFarms) : farmsList(inactiveFarms)
-  }
+  const loadMoreRef = useRef<HTMLDivElement>(null)
+
+  const [numberOfItemsToShow, setNumberOfItemsToShow] = useState(12)
```

Comment:
Avoid magic numbers in the code. Set this in a variable at the top of the file, with a wisely picked name please.

Reply:
Set it to a constant named `NUMBER_OF_FARMS_VISIBLE`. Also did some more renaming, so the code is a bit easier to understand.
Thanks a lot for the feedback!